life/modernized/02-life-of-gilbert-latey.adoc

```diff
@@ -1809,10 +1867,15 @@ I shall here add one account more,
 it being an application drawn up and signed by Friends, and delivered to King William,
 by Gilbert and some Friends appointed; and is as follows:
 
+[.embedded-content-document.address]
+--
+
+[.letter-heading]
```

Comment:
This is another one of those weirdly rendered letters that I couldn't remember what we did exactly before.